crates/bench-api/Cargo.toml

```diff
@@ -37,3 +37,4 @@ wat = { workspace = true }
 [features]
 default = ["shuffling-allocator", "wasi-nn"]
 wasi-nn = ["wasmtime-wasi-nn"]
+winch = ["wasmtime/winch"]
```

Comment:
I decided to go with a feature here, given that `winch` is still not a default wasmtime feature, however I wonder if it might make sense to unconditionally enable the `winch` feature in this crate.